tests/src/rules/dynamic-import-chunkname.js

```diff
@@ -1299,13 +1326,21 @@ context('TypeScript', () => {
             /* webpackPrefetch: true */
             /* webpackPreload: true */
             /* webpackIgnore: false */
-            /* webpackMode: "eager" */
+            /* webpackMode: "lazy" */
```

Comment:
Ditto here, this test is checking for webpack comment correctness, not eager mode, so changing this to "lazy"